src/instructlab/model/phased_training.py

```diff
@@ -0,0 +1,140 @@
+# Standard
```

Comment:
Would it make sense to identify a given training run with a unique ID? This ID could be left in the journal or the journal's name.

I think that user shouldn't have to think about the journal and that if we run multiple trainings without specifying a journal we should avoid a collision.

What do you think?

Reply:
Yeah that's a good idea

Reply:
added